instaloader/instaloadercontext.py

```diff
@@ -528,6 +528,25 @@ def get_and_write_raw(self, url: str, filename: str) -> None:
         :raises ConnectionException: When download repeatedly failed."""
         self.write_raw(self.get_raw(url), filename)
 
+    def head(self, url: str, allow_redirects: Optional[bool] = None) -> requests.Response:
+        """HEAD a URL anonymously.
+
+        :raises QueryReturnedNotFoundException: When the server responds with a 404.
+        :raises QueryReturnedForbiddenException: When the server responds with a 403.
+        :raises ConnectionException: When request failed."""
```

Comment:
```suggestion
        :raises ConnectionException: When request failed.
        
        .. versionadded:: 4.7.6
        """
```

(add *versionadded* directive)